src/App.tsx

```diff
@@ -36,14 +52,48 @@ function App() {
 								<Route path="/" element={<LandingPage />} />
 								<Route
 									path="/dashboard"
-									element={<DashboardPage />}
+									element={
+										<ProtectedRoute>
```

Comment:
Try do this.
```
<Routes>
  <ProtectedRoutes>
     <Route path =... />
     <Route path=... />
     <Route path=.... /> 
  </ProtectedRoutes>
</Routes>
```

Reply:
Sorry i typed wrong.

<Routes>
  <Route element = {<ProtectedRoutes/>}>
     <Route path =... />
     <Route path=... />
     <Route path=.... /> 
  </Routes>
</Routes>

Reply:
this won't work because the direct children of `<Routes>` has to be a `<Route>` component